modules/swagger-core/src/main/java/io/swagger/core/filter/AbstractSpecFilter.java

```diff
@@ -1,44 +1,40 @@
 package io.swagger.core.filter;
 
 import io.swagger.model.ApiDescription;
+import io.swagger.oas.models.OpenAPI;
 import io.swagger.oas.models.Operation;
+import io.swagger.oas.models.PathItem;
 import io.swagger.oas.models.media.Schema;
 import io.swagger.oas.models.parameters.Parameter;
 
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 
-public abstract class AbstractSpecFilter implements SwaggerSpecFilter {
-    public boolean isOperationAllowed(
-            Operation operation,
-            ApiDescription api,
-            Map<String, List<String>> params,
-            Map<String, String> cookies,
-            Map<String, List<String>> headers) {
-        return true;
+public abstract class AbstractSpecFilter implements OpenAPISpecFilter {
```

Comment:
should add a "removeBrokenReferences" to interface and abstract class, similar to 
https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-core/src/main/java/io/swagger/core/filter/AbstractSpecFilter.java#L42